examples/test.cpp

```diff
@@ -1,6 +1,43 @@
-#include "../include/featurelib.h"
+#include "../include/feature_lib.h"
+#include <iostream>
+#include <opencv2/core.hpp>
+#include <opencv2/imgcodecs.hpp>
```

Comment:
Rearrange includes as the code style [suggests](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)

Reply:
done